force-app/main/default/classes/SOQL.cls

```diff
@@ -66,6 +84,11 @@ public with sharing class SOQL {
     //[public | private ] [static] data_type method_name (input parameters) {
     //  The body of the method
     //}
+    public static List<Opportunity> getClosedWonOpportunityCountByAccount(Id accountId) {
+        List<Opportunity> ClosedWonOpportunity = [SELECT Id, Name, StageName, AccountId, CloseDate, Amount FROM Opportunity
+        WHERE AccountId = :accountId AND StageName = 'Closed Won'];
```

Comment:
This is where you'll want to consider formatting vertically as your queries get longer and more complex:
```suggestion
        List<Opportunity> opps = [
                SELECT 
                        Id,
                        Name,
                        StageName,
                        AccountId,
                        CloseDate,
                        Amount 
                FROM Opportunity 
                WHERE AccountId = :accountId 
                AND StageName = 'Closed Won'
        ];
```